src/services/WolfDen.Infrastructure/Configuration/LeaveRequestConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using WolfDen.Domain.Entity;
+
+namespace WolfDen.Infrastructure.Configuration
+{
+    public class LeaveRequestConfiguration:IEntityTypeConfiguration<LeaveRequest>
+    {
+        public void Configure(EntityTypeBuilder<LeaveRequest> builder)
+        {
+            builder.Property(x => x.Id).HasColumnName("LeaveRequestId");
+            builder.Property(x => x.Description).HasMaxLength(1000);
+            builder.HasOne(x => x.Employee).WithMany().HasForeignKey(x => x.EmployeeId).OnDelete(DeleteBehavior.Restrict); 
```

Comment:
you can either use WillCascadeOnDelete(false) also

Reply:
okay